returnn/datasets/meta.py

```diff
@@ -1486,7 +1491,7 @@ def get_seq_len(i):
             sub_seq_list.extend(sub_seq_tags)
         assert sub_seq_idx == len(sub_seq_list) and len(seq_list) == len(sub_seq_idxs)
         self.cur_sub_seq_idxs = sub_seq_idxs
-        return self.sub_dataset.init_seq_order(seq_list=sub_seq_list)
+        return self.sub_dataset.init_seq_order(epoch=epoch, seq_list=sub_seq_list)
```

Comment:
I wonder a bit, why this change? It should be ok, but actually also should not matter? Or was there a problem? I'm just curious.

Reply:
The problem is it could be incompatible with some sub_dataset like `MetaDataset`, where init_seq_order would not be actually executed if `epoch` stays `None`, maybe there are other workarounds?

Reply:
Ah yea, good example. Yes this is definitely more correct.